source/server-selection/server-selection.rst

```diff
@@ -1157,7 +1152,7 @@ individual read operation.
 Reference Implementation
 ========================
 
-The single-threaded reference implementation is the Perl master branch (work
+The single-threaded reference implementation is the Perl main branch (work
```

Comment:
This branch does not exist in https://github.com/mongodb-labs/mongo-perl-driver/branches, where would one find it?

Reply:
We are not renaming git branches yet. That's being left for a future project. I should probably have left this alone. I can revert this change.

Reply:
Switched back from main to master.